x-pack/plugin/sql/build.gradle

```diff
@@ -51,6 +51,16 @@ bundlePlugin {
     }
 }
 
+// add all sub-projects of the qa sub-project
+gradle.projectsEvaluated {
```

Comment:
We don't have a need for `projectEvaluated` here. 
`check.dependsOn` can take a closure which is evaluated at the right time,
or you can use `project.subprojects.matches` instead of `findAll`.

`project.subprojects.find` can be simplified by referencing the project directly as `project(project.path + ':qa')`

Reply:
Would it be OK with you if I merge this and the graph one as is and then made the simplifications in a follow up change? I knew it could be simplified but didn't want to go and change the other instances of the code in this PR. I didn't know `dependsOn` could take a closure though!